src/OrchardCore/OrchardCore.ResourceManagement/TagHelpers/StyleTagHelper.cs

```diff
@@ -68,6 +63,11 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
                     setting.AtLocation(ResourceLocation.Head);
                 }
 
+                foreach (var attribute in output.Attributes)
+                {
+                    setting.SetAttribute(attribute.Name, attribute.Value.ToString());
```

Comment:
Just for curiosity why moving this here from the above

Reply:
Literally no reason @jtkech I'd moved the above `AtLocation` to later when testing, but when i didn't need it later, I moved it back, but forgot where it was originally ;)

I just moved it back